queries.py

```diff
@@ -1052,17 +1059,37 @@
     }
 """
 
+GET_PURCHASE_LINE_ETAG = """
+query PurchaseOrderLine($id:ID!) {
+    purchaseOrderLine(id:$id)
+    {
+        _etag
+    }
+}
+
+"""
+
 GET_PURCHASE_LINES = """ 
     query PurchaseOrderLines($filterss: PurchaseOrderLinesInputFilters) {
         purchaseOrderLines(filters: $filterss) {
             edges {
                 node {
                     id
                     _etag
+                   partInventories {
+                    installed
```

Comment:
This indentation doesn't look quite right, partInventories should line up with same indentation as `_etag`